htdocs/takepos/index.php

```diff
@@ -73,7 +73,8 @@
 	// We will recalculate amount for foreign currency at next call of invoice.php when $_SESSION["takeposcustomercurrency"] differs from invoice->multicurrency_code.
 }
 
-$_SESSION["urlfrom"] = '/takepos/index.php';
+// TODO it creates side effects
+//$_SESSION["urlfrom"] = '/takepos/index.php';
```

Comment:
This seems to introduce a problem with the logout from takepos. 
After this change when we logout from Takepos, we go back on login (good), but when we login, we do not reache the takepos page again. 
Instead we reach the backoffice (with sometime access error when users are allowed to use the takepos only).

What was wrong with old code ?